src/tests/system/tests/test_ipa_trusts.py

```diff
@@ -178,3 +178,46 @@ def test_ipa_trusts__aduser_membership_after_HBAC(ipa: IPA, trusted: GenericADPr
     result = ipa.tools.id(aduser_fqn)
     assert result is not None, "User is not found"
     assert result.memberof(posix_group.name), f"User lost membership in '{posix_group.name}' after HBAC update."
+
+
+@pytest.mark.importance("low")
+@pytest.mark.ticket(jira="RHEL-94545", gh=8048)
+@pytest.mark.topology(KnownTopologyGroup.IPATrust)
+def test_ipa_trusts__lookup_private_group_with_username_override(ipa: IPA, trusted: GenericADProvider):
```

Comment:
Hi,

since we do not have many tests for id-overrides what do you think of adding tests like '__lookup_user_private_group(_without_username_override)', '__lookup_ad_group' and '__lookup_ad_group_with_groupname_override'?

Additionally, it is expected that for overwritten names the lookup with the original name is still working but will return the object with the new name. I think this should be part of the test as well.

bye,
Sumit

Reply:
Fixed, will just wait for CI now.

Reply:
Thanks